content/04.results.md

```diff
@@ -30,149 +30,200 @@ Taken together, the results suggest that while bioRxiv has experienced dramatic
+We examined whether or not the number of versions of a preprint was associated with a change in the time to publish.
+We found that preprints with more versions generally took longer to publish (Figure {@fig:publication_delay_panels}B).
+Linear regression comparing the time to publish with the number of versions revealed an increasing relationship, with each version being associated with an additional 51 days before publication.
+This time period seems broadly compatible with the amount of time it would take to receive reviews and revise a manuscript, suggesting that many authors may be updating their preprints in response to peer reviews or other external feedback.
+
+We next used the full text content to examine the extent to which the magnitude of change to the text between the preprint and peer reviewed publication was associated with a change in the time to publication.
+We used distance in the document embedding space to quantify the amount of change to the text.
+We first established a baseline by selecting pairs of random preprints in the author-selected bioinformatics category and found a mean embedding distance between random pairs of 5.068.
+Preprints that were further in the embedding space from their corresponding peer reviewed publication took longer to publish, with each unit in the embedding space corresponding to approximately sixteen additional days (Figure {@fig:publication_delay_panels}A).
+The number of version and document distance effects, taken together, support a model where preprints that are reviewed more or that require larger revisions take longer to publish.
+
+### Preprints with similar document embeddings share publication venues
+
+We hypothesized that document embedding might capture information associated with the eventual publication venue.
+We used the annotated preprint-publication pairs as a fully held out test set.
+We embedded all PMC papers except those annotated to have a corresponding preprint into the bioRxiv embedding space.
+For journals, we averaged the document representation of all papers in the journal to produce a journal centroid.
+We used a k-nearest neighbor classifier trained on the remaining PMC documents to identify the most similar published papers and assessed performance - the extent to which the journal a paper appeared in the returned list of ten neighbors - through cross validation and the held out test set (Supplemental Figure {@fig:knn_auc}).
+We found an enrichment of papers published within the same journal among the neighbors of the target paper.
+
+We developed an online app that returns the nearest published papers and journals that are nearest to a preprint.
+Users supply DOIs from _bioRxiv_ or _medRxiv_. <!-- we need to be consistent in how these are represented - should they always be italicized? it feels like a journal name would be - perhaps see what the bioRxiv folks did in their own paper. -->

```

Comment:
```suggestion
Users supply DOIs from bioRxiv or medRxiv. 
```

Reply:
bioRxiv wasn't italicized in the bioRxiv folks' paper but journals like PLOS genetics and Proceedings of the Royal Society were. Since bioRxiv isn't a journal, I don't believe we need to italicize it.